prow/plugins/trigger/pr.go

```diff
@@ -30,43 +30,43 @@ const (
 	needsOkToTest = "needs-ok-to-test"
 )
 
-func handlePR(c client, trustedOrg, joinOrgURL string, pr github.PullRequestEvent) error {
-	author := pr.PullRequest.User.Login
+func handlePR(c client, trigger *plugins.Trigger, pr github.PullRequestEvent) error {
+	author, org, repo := orgRepoAuthor(pr.PullRequest)
```

Comment:
These variables are not assigned correctly. They should be `org, repo, author`. Unit tests should catch things like this, but apparently we don't have any for `handlePR`, just `handleIC`...

Reply:
Done